scene/gui/tree.cpp

```diff
@@ -777,6 +800,7 @@ TreeItem *TreeItem::create_child(int p_index) {
 	}
 
 	ti->parent = this;
+	ti->parent_visible_in_tree = visible && parent_visible_in_tree;
```

Comment:
```suggestion
	ti->parent_visible_in_tree = is_visible_in_tree();
```
It's fine to refer to private properties like this, but not much in complex conditions. If `is_visible_in_tree()` ever changes, this will be wrong.